src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateActionPlanTest.kt

```diff
@@ -280,4 +280,31 @@ class CreateActionPlanTest : IntegrationTestBase() {
     assertThat(reviewScheduleHistoryRepository.findAllByReference(reviewScheduleReference)).isNotNull
     assertThat(reviewScheduleHistoryRepository.findAll()).size().isEqualTo(1)
   }
+
+  @Test
+  fun `should create action plan and not create initial review schedule given prisoner already has an induction created before the action plan, but is an unsupported sentence type for the release schedule`() {
```

Comment:
This, and the test in the next class, are the 2 crucial tests, that test the behaviour we are trying to fix.